### ISSUE TITLE
Check/Kick doesn't work

### ISSUE THREAD

**Issue by crashzk** [azalty/sm-no-dupe-account#34]
**Describe the bug**
Check/Kick doesn't work, player doesn't have the requirements I configured and even then it's not being kicked from the server.

**Error logs**
```
L 08/28/2022 - 08:41:02: [SM] Exception reported: Language phrase "Kicked_Bans_Total" not found (arg 4)
L 08/28/2022 - 08:41:02: [SM] Blaming: no_dupe_account.smx
L 08/28/2022 - 08:41:02: [SM] Call stack trace:
L 08/28/2022 - 08:41:02: [SM]   [0] KickClient
L 08/28/2022 - 08:41:02: [SM]   [1] Line 2035, D:\Perso\expbox\RELEASE\no-dupe-account\1.5.0\sm-no-dupe-account\scripting\no_dupe_account.sp::OnCheckSteamBansResponse
```
In the case of a translation error, there is no such sentence in the translation file.

https://github.com/azalty/sm-no-dupe-account/blob/cb5183e1b874cd515f7527713c3dc38a37906512/scripting/no_dupe_account.sp#L2033-L2036

Player that would not have the requirements would be **[Thorfinn](https://steamcommunity.com/profiles/76561199203450450)** - **`STEAM_1:0:621592361`**.

![image](https://user-images.githubusercontent.com/32937653/187091648-cde94bff-90e5-4623-9ce7-9e2ffb6ec823.png)

Here is the complete plugin with the configuration and translations I am currently using.

As I wanted the plugin to work, it will check the player, if it does not pass any of the requirements it is kicked, even if it has passed all the other requirements if it does not pass the minimum hours requirements in CSGO it does not pass and it is kicked from the server.

Player in question does not have server privileges, does not have the required hours, 120 hours in CSGO.

![image](https://user-images.githubusercontent.com/32937653/187091924-624a9b9b-706c-4ab5-8d54-286e7f70287f.png)

![image](https://user-images.githubusercontent.com/32937653/187091933-fb61461b-3c20-4e4d-8dcc-fd83ad3bdea1.png)

**Comment by crashzk** [comment]
Anyway, error would be, player even not having the requirements is not kicked from the server.

In the case of the player I mentioned above, it would be on time, he needs to have 120 in CSGO to play on the server, he doesn't have this requirement and still manages to enter.

**Comment by azalty** [comment]
@crashzk watch out you leaked your steam api key and discord webhook

**Comment by azalty** [comment]
Thanks for reporting, the player not being kicked is due to the missing translation. My bad :-/

Adding it in a commit, I'll do a release as well.

The `database status: Checked` doesn't mean the player has been allowed, it only means their data has been found and loaded from the database. If more checks are needed, the plugin will wait for the player to connect to check their csgo level or profile coin for example.

**Comment by azalty** [comment]
Fixed in 1.5.1

**Comment by azalty** [comment]
Another problem we found was that their config looked like that:

```
// (Requires SteamAPI Key)
// 0 = disabled
// any integer = is a user check that fails if he has less mins in playtime than asked or has private hours
// any negative integer = same as positive, but is not a check and will kick user
// -
// Default: "120"
nda_playtime "-7200"    // 120 Horas
```
We found out that comments right after the value (`nda_playtime "-7200"`) break things. We found out that the cvar value was `""-7200"    // 120 Horas"` due to this. Weird bug.
